bciers/libs/components/src/navigation/Bread.tsx

```diff
@@ -67,6 +41,20 @@ export default function Bread({
   const searchParams = useSearchParams();
   const crumbTitles = Object.fromEntries(searchParams.entries());
 
+  // 🛠️ Function to check if a link is the last link
+  const isLastBreadcrumbItem = (link: string, index: number) => {
+    const lastLinkValues = ["register-an-operation"];
```

Comment:
For cleanliness sake should we make this an argument that we pass in the `layout` similar to `defaultLinks`?

Reply:
@marcellmueller 

A good thought...
Although, I see it as more of a static list...but, of course, I am biased by a wee anti props drilling attitude.